lib/scan_result.dart

```diff
@@ -1,8 +1,8 @@
 part of flutter_ble_lib;
 
 abstract class _ScanResultMetadata {
-  static const String id = "id";
-  static const String name = "name";
+//  static const String id = "id";
```

Comment:
shouldn't the fields be removed?

Reply:
I dunno. I just wanted to get rid of [these static analysis](https://pub.dev/packages/flutter_ble_lib/score) issues. I lack context.

Reply:
They should be removed, they're used for passing peripheral information, but it's deserialized using `Peripheral.fromJson` and its own metadata.